internal/usecase/client/rbac/wrapper.go

```diff
@@ -113,6 +115,7 @@ func (c *rbacWrapper) retrieveACL(ctx context.Context) ([]string, error) {
 
 	limit = limitDefault
 	offset = 0
+	permission := []string{}
```

Comment:
If there's another reason to amend this PR, then sure, it would be nice to `s/permission/permissions/`.  But otherwise, I don't think it's a show-stopper.